### PR TITLE
Added Image Captioning feature

### PR DESCRIPTION
**Pull request by Pravz-149**

### CODE REVIEW

File: usellm.org/package-lock.json
Comment (BirajCoder) [inline usellm.org/package-lock.json:1]: Please remove the changes in this file

File: usellm.org/package.json
Comment (BirajCoder) [inline usellm.org/package.json:1]: Please remove the changes in this file

File: usellm.org/config/demos.ts [hunk 1]
```diff
@@ -72,4 +73,11 @@ export const HOME_PAGE_DEMOS = [
     sourceUrl:
       "https://github.com/usellm/usellm/blob/main/usellm.org/app/demo/clone-voice/page.tsx",
   },
+  {
+    name: "Image Captioning",
+    id: "image-captioning",
+    component: ImageCaptioning,
+    sourceUrl:
+      "https://github.com/usellm/usellm/blob/main/usellm.org/app/demo/image-captioning/page.tsx",
+  },
```
Comment (BirajCoder) [inline usellm.org/config/demos.ts:82]: Please pull the latest changes and resolve the conflicts

File: usellm.org/package-lock.json
Comment (Pravz-149) [reply usellm.org/package-lock.json:1]: Done

File: usellm.org/package.json
Comment (Pravz-149) [reply usellm.org/package.json:1]: Done

**Pull request closed without merging**